### DISCUSSION TITLE
Integrations with other libraries and tools

### DISCUSSION THREAD

**Discussion by mootari** [rusq/slackdump#135] [General]
This discussion tracks integration efforts between slackdump and other libraries, primarily by the libraries' authors.

**Comment by mootari** [comment]
#### [jackellenberger/emojme](https://github.com/jackellenberger/emojme)

Issue: [[Feature Request] use slackdump's playwright auth strategy](https://github.com/jackellenberger/emojme/issues/73)

The author has expressed interest to either replicate slackdump's authentication mechanics or to integrate slackdump directly.

**Reply by rusq** [reply]
https://github.com/mattermost/mmetl/issues/20

The author enquired about the license type, however, deeply disappointed by the lack of reply for 15 days, closed the issue in frustration.

**Comment by rusq** [comment]
https://github.com/slack-go/slack/pulls/rusq PRs open in slack-go/slack library.
